build-setup/windows.md

```diff
@@ -4,10 +4,10 @@ These instructions are for setting up a Windows build machine from scratch.
 
 For subsequent machines, the setup can be simplified by copying components from the first machine - see [windows_simple.md](windows_simple.md).
 
-The build system assumes Windows itself is installed on the `C:` drive. If this is not the case on your machine, the Makefiles will not work.
-
 On Windows, the build system runs within Git for Windows. This provides a bash shell that enables our Unix build system to work on Windows without alteration.
 
+The build system assumes Windows itself is installed on the `/C` drive. If this is not the case on your machine, set the `ROOT` environment variable to the appropriate drive.
+
 If you use Gradle to build Machine Learning you do not need to work in Git Bash, as Gradle tasks will run it when necessary.  In order to develop interactively on Windows, you must work in Git Bash.  You will need the following environment variables to be defined:
```

Comment:
> you must work in Git Bash

Like being tethered to the `C:` drive, it would be nice to avoid the need to work in Git Bash.  CMake generating Visual Studio project files _should_ make it possible to work outside of Git Bash.

It would be nice to do an exercise to list all the reasons why Git Bash is still needed as an integral part of the build system and then figure out how each reason could be avoided by using CMake functionality instead.

But this can be left for a future PR.